astropy/io/fits/hdu/hdulist.py

```diff
@@ -105,6 +105,22 @@ def fitsopen(name, mode='readonly', memmap=None, save_backup=False,
 
         .. versionadded:: 4.2
 
+    use_fsspec : bool, optional
+        Is the fsspec library to be used to open the file?
+        Defaults to `False` unless ``name`` starts with prefix "s3://"
+        (Amazon S3 storage) or prefix "gcs://" (Google Cloud Storage).
+        This feature requires the optional ``fsspec`` package to be installed.
+        In addition, Amazon S3 files also require the ``s3fs`` optional
+        dependency, and Google Cloud files require ``gcsfs``.
+
+        .. versionadded:: 5.1
```

Comment:
we're already in the 5.2 dev cycle

```suggestion
        .. versionadded:: 5.2
```

Reply:
Merged the suggestion! :+1: